src/perl/Permabit/Statistics/Makefile

```diff
@@ -5,4 +5,3 @@ MANPREFIX := Permabit::Statistics::
 include $(ROOT)/Makefile.common
```

Comment:
It kind of seems like we don't need the perl/Permabit/Statistics makefile at all, now. Can we just get rid of it? Maybe as part of the second commit...

Reply:
I was wondering if we should get rid of it.  I left it there in case we think people want to add things in the future.  I will remove it.